Sources/StreamChatUI/Utils/CountdownTracker.swift

```diff
@@ -0,0 +1,81 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
```

Comment:
We don't need to import UIKit in this case
```suggestion
import Foundation
```

Also, it is probably a good idea to move the `CooldownTracker` and the `StreamTimer`'s to the `StreamChat` framework, so that we can use this in the SwiftUI SDK as well CC @martinmitrevski

Reply:
Changed, and moved files to StreamChat